CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Developer guidelines
+
+<!-- vim-markdown-toc GFM -->
+
+- [Development](#development)
+  - [Dev environment set up](#dev-environment-set-up)
+- [Pull requests](#pull-requests)
+- [Merging to main](#merging-to-main)
+  - [Stages: Draft and Ready for review](#stages-draft-and-ready-for-review)
+
+<!-- vim-markdown-toc -->
+
+## Development
+
+### Dev environment set up
```

Comment:
This is probably obvious, but we could add that `npm install` must be run first

Reply:
why not ig